ZeekPluginCommon.cmake

```diff
@@ -185,16 +102,32 @@ macro(zeek_plugin_end)
     endif ()
 endmacro()
 
-# This is needed to support legacy Bro plugins.
-macro(bro_plugin_end)
-    zeek_plugin_end(${ARGV})
+# -- wrappers for legacy bro_* function names ----------------------------------
+
+macro(bro_plugin_begin)
+    zeek_plugin_begin(${ARGV})
```

Comment:
Should we add deprecation messages or something on these so that people know we're going to remove them eventually?

Reply:
Added.

Reply:
How about for the rest of them? 😃

Reply:
Wouldn't that be a bit overkill. 🙂

All legacy plugins will use `bro_plugin_begin` and thus see the warning. Deprecating all of them would only spam the log with redundant information.

Reply:
Good point.